docusaurus/docs/create-a-plugin/develop-a-plugin/panel-properties.md

```diff
@@ -0,0 +1,106 @@
+---
+id: subscribe-events
+title: Subscribe to Grafana events
+description: Work with the event bus to subscribe to Grafana application events
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - panel
+  - properties
+  - eventbus
+  - event bus
+  - subscribe
+  - events
+sidebar_position: 5
+---
+
+# Subscribe to Grafana events
+
+If you’re building a [panel plugin](../../introduction/plugin-types-usage.md#panel-visualization-plugins), Grafana already provides the data you need to configure and render your panel. If not, you can usually retrieve the information you need from any of the runtime services, like `BackendSrv` and `TemplateSrv`.
+
+However, in some cases you may want your plugin to react to changes that occur outside of your plugin. For example, you may want your plugin to react when the user zooms in or out of another panel. In this guide, you’ll learn how to make your plugin react to events in Grafana.
+
+:::tip
+
+For a step-by-step guide to making your own panel plugin, refer to our [Tutorial for panel plugins](../../tutorials/build-a-panel-plugin.md).
+
+:::
+
+## Access the event bus
+
+The event bus for subscribing to Grafana events is located on the [`PanelProps`](https://github.com/grafana/grafana/blob/57960148e47e4d82e899dbfa3cb9b2d474ad56dc/packages/grafana-data/src/types/panel.ts#L74-L122) interface. This interface exposes runtime information about the panel, such as the panel's dimensions and timing measurements. Refer to the code comments for the property definitions.
+
+You can access the event bus and other panel properties through the `props` argument of your plugin. For example:
+
+```js title="src/components/SimplePanel.tsx"
+export const SimplePanel: React.FC<Props> = ({ options, data, width, height }) => {
```

Comment:
non-blocking, its a little odd that the eventbus _isn't_ one of the properties listed in this example given the above.

Reply:
non-blocking for now